### PR TITLE
priority pid tests should match on processes

### PR DESCRIPTION
**Pull request by kannon92**
#### What type of PR is this?
/kind failing-test

#### What this PR does / why we need it:
PID tests are flaky. Maybe this can address this.

PodDisruptionCondition test for PIDs are less flaky so I wanted to match the process amount.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by kannon92** [comment]
/cc @mimowo 

Any reason why these were different?

**Comment by mimowo** [comment]
> /cc @mimowo 
> 
> Any reason why these were different? 

i only vaguely  remember I was running the tests locally in a loop and I was finding them flaky at the lower count.

File: test/e2e_node/eviction_test.go [hunk 1]
```diff
@@ -475,6 +475,7 @@ var _ = SIGDescribe("PriorityPidEvictionOrdering", framework.WithSlow(), framewo
 
 	highPriorityClassName := f.BaseName + "-high-priority"
 	highPriority := int32(999999999)
+	processes := 30000
```
Comment (mimowo) [inline test/e2e_node/eviction_test.go:478]: Maybe name it `forBombProcessesCount` to make the intention clear. Alternatively, to make sure the same value is used here as for PodDisruptionConditions I would suggest to introduce a function 
`forkBombPod(name string)` calling  `pidConsumingPod(name, 30000)`. And use the function in all places.

Comment (kannon92) [reply test/e2e_node/eviction_test.go:478]: These tests take a long time so I'll update it once it finishes running.

**Comment by kannon92** [comment]
Doesn't look like this fixes it.

I think 0 process stats is affecting both crio/containerd.

#122927

**Comment by kannon92** [comment]
Real reason for the flakiness is https://github.com/kubernetes/kubernetes/issues/115215.

Added a test to confirm that cadvisor is failing to collect stats on processes (always returns 0 for both crio/containerd).

**Comment by kannon92** [comment]
/close
@pacoxu merged this one into his and I think thats a good idea

**Pull request closed without merging**